### PR TITLE
setup.py

### PR DESCRIPTION
**Pull request by ric-evans**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    kwargs["install_requires"] = install_requires
+    kwargs["dependency_links"] = dependency_links
+    kwargs["zip_safe"] = False
+
+
+setup(
+    name=PACKAGE,
+    scripts=glob.glob("bin/*"),
+    packages=["krs", "resources"],
```
Comment (dsschult) [inline setup.py:73]: Also need `user_mgmt` package.  And I'm not sure `resources` should be a package.  Maybe it just needs to be named something else (or rather, that script needs to be moved to a better directory).

**Review by dsschult — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (ric-evans) [reply setup.py:73]: Would this make more sense in `krs` or `user_mgmt`?

Comment (dsschult) [reply setup.py]: I think a new directory, maybe `krs_setup`?

Comment (dsschult) [reply setup.py]: or `keycloak_setup`

**Review by dsschult — APPROVED** [review]

**Pull request merged**